packages/plugins/integration-zapier/src/lib/zapier.controller.ts

```
@@ -0,0 +1,164 @@
+		try {
+			return await this._zapierService.addIntegration(body);
+		} catch (error) {
+			this.logger.error('Failed to create Zapier integration', error);
+			throw new InternalServerErrorException('Failed to create Zapier integration');
+		}
+	}
+
+	/**
+	 * Helper method to validate Zapier token
+	 */
+	private validateToken(token: string, isAction: boolean = false): void {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)